### DISCUSSION TITLE
OpenSUSE Installation correction

### DISCUSSION THREAD

**Discussion by sentairanger** [opentofu/opentofu#1176] [General]
I successfully installed Open Tofu on OpenSUSE, however this line caused an issue when I first attempted to install.

`./install-opentofu.sh --install-method rpm`

I resolved this by adding sudo. This should be updated to reflect that.

**Comment by ghost** [comment]
Hey @sentairanger thank you for reporting this. This is a bug in the install script, it should request sudo itself if it has to, the rest of the script should run without root privileges. Issue opened here: https://github.com/opentofu/get.opentofu.org/issues/19 and here's the fix for it: https://github.com/opentofu/get.opentofu.org/pull/20

**Update:** This issue should now be fixed, please test and let us know if it works.

**Reply by sentairanger** [reply]
I tested this on a Open SUSE VM and it worked without issue thank you.
